src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import { getPageNumbers, getNumbers } from '../../utils';
+
+type PaginationProps = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page: number) => void,
+  startIndex: number,
+  endIndex: number,
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+  startIndex,
+  endIndex,
+}: PaginationProps) => {
+  const pageNumbers = getPageNumbers(total, perPage);
+  const items = getNumbers(1, total).map(n => `Item ${n}`);
+  const isFirstPage = currentPage === 1;
+  const isLastPage = currentPage === pageNumbers.length;
+
+  return (
+    <>
```

Comment:
redundant [fragment](https://react.dev/reference/react/Fragment#usage)

Reply:
Not fixed